csharp/src/ProtocolBuffers/Google.Protobuf.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Google.Protobuf</id>
+    <title>Google Protocol Buffers C#</title>
+    <summary>C# runtime library for Protocol Buffers - Google's data interchange format.</summary>
+    <description>See project site for more info.</description>
+    <version>3.0.0-experimental</version>
+    <authors>Google Inc.</authors>
+    <owners>protobuf-packages</owners>
```

Comment:
this is probably overriden anyway when you are uploading the package to nuget, so currently it only serves as documentation under which nuget account we want to published it (I've already created that account and reserved the package name).